server/src/main/java/de/uftos/services/ConstraintInstanceService.java

```diff
@@ -45,7 +46,8 @@ public ConstraintInstance create(String signatureId, ConstraintInstanceRequestDt
    * @param pageable    contains the parameters for the page.
    * @return the page of the entries fitting the parameters.
    */
-  public Page<ConstraintInstance> get(String signatureId, Pageable pageable) {
+  public Page<ConstraintInstance> get(String signatureId, Pageable pageable,
+                                      Optional<String> argument) {
     return this.repository.findAll(pageable);
```

Comment:
```suggestion
    return this.repository.findAll(pageable, argument);
```

Reply:
That's not how this works. Also, that's logic, so let's skip it for now